integration_tests/integration/barcode/createBarcodeAddressImage.spec.ts

```diff
@@ -5,6 +5,7 @@ import ReviewRecipientsPage from '../../pages/barcode/reviewRecipients'
 
 context('Create Barcode Image', () => {
   beforeEach(() => {
+    cy.task('reset')
```

Comment:
Doing the reset a in `goTo` method was restricting - couldn't set up a stub before calling that goTo method - better for each test to  perform the reset.